syncstorage/views/__init__.py

```diff
@@ -461,7 +463,9 @@ def get_item(request):
     userid = request.validated["userid"]
     collection = request.validated["collection"]
     item = request.validated["item"]
-    return storage.get_item(userid, collection, item)
+    bso = storage.get_item(userid, collection, item)
+    bso.pop("ttl", None)
+    return bso
```

Comment:
Although we return the "ttl" field so that it can be used by memcached, we don't want to return it to the client.  This change ensure it's removed at the network boundary.